pex/vendor/__init__.py

```diff
@@ -88,37 +90,9 @@ def iter_vendor_specs():
   :return: An iterator over specs of all vendored code.
   :rtype: :class:`collection.Iterator` of :class:`VendorSpec`
   """
-  yield VendorSpec.create('pip==19.3.1', rewrite=False)
-  yield VendorSpec.create('setuptools==41.6.0')
-  yield VendorSpec.create('wheel==0.33.6')
-
-
-def _vendored_dists(distributions):
```

Comment:
N.B.: `setup_interpreter` and its helper `_vendored_dists` needed the removed `VendorSpec.version` property but were unused.